pkgs/http_client_conformance_tests/lib/src/redirect_tests.dart

```diff
@@ -76,24 +56,23 @@ void testRedirect(Client client) async {
     });
 
     test('too many redirects', () async {
-      final request = Request('GET', Uri.http('localhost:${server.port}', '/6'))
+      final request = Request('GET', Uri.http(host, '/6'))
         ..followRedirects = true
         ..maxRedirects = 5;
       expect(
           client.send(request),
           throwsA(isA<ClientException>()
               .having((e) => e.message, 'message', 'Redirect limit exceeded')));
-    });
+    }, onPlatform: {'browser': const Skip('not passing')});
```

Comment:
[nit] can move the const up a level
```suggestion
    }, onPlatform: const {'browser': Skip('not passing')});
```